docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+
+.. contents::
+
+Installing MUSE
+===============
+
+Recommended installation
+------------------------
+
+To help you installing MUSE in your system we will follow these steps:
+
+- `Launching a terminal`_: Needed to both install and run MUSE.
+- `Installing a compatible Python version`_: At the moment, MUSE works with Python 3.8 and 3.9.
+- `Installing pipx`_: A Python application manager that facilitates installing, keeping applications updated and run them in their own isolated environments.
+- `Installing MUSE itself`_
+
+In the following sections, we will guide you step by step in configuring your system so it can run MUSE. At the end, we include alternative ways of installing MUSE if this method does not work for any reason, for advanced users and for developers of MUSE.
+
+.. note::
+
```

Comment:
It worked for me in a brand new Windows 11 computer - I didn't even need to disable the alias. However, as indicated in the note, people should read the rest of the instructions. This is here to set expectations of what they will need to run, ultimately, and also because not only total novices will try to install MUSE. They will know what to do out of these minimal instructions.